envs/example/ci-full/group_vars/all.yml

```diff
@@ -1,17 +1,33 @@
 ---
-stack_env: example_ci-full
+stack_env: ci-full
+ursula_os: 'ubuntu'
+undercloud_cidr:
+  - cidr: 192.168.0.0/22
 
 state_path_base: /opt/stack/data
 
-neutron:
-  enable_external_interface: True
-  lbaas:
-    enabled: True
+
+hostname_fqdn: "{{ ansible_fqdn }}"
+hostname_short: "{{ hostname_fqdn.split('.')|first }}"
+
+floating_ip: "{{ hostvars[groups['controller'][0]][primary_interface]['ipv4']['address'] }}"
 
 keystone:
   ldap_domain:
     enabled: True
     domain: users
 
+neutron:
+  enable_external_interface: True
+  path_mtu: 1400
+  l3ha:
+    enabled: True
+  lbaas:
+    enabled: True
+
+heat:
+  enabled: True
+  debug: True
```

Comment:
This is not needed. Default-2.0.yml has this set
https://github.com/blueboxgroup/ursula/blob/RELEASE-3.1.x/envs/example/defaults-2.0.yml#L572